src/aggregators/operations-aggregator/operations-aggregator-helpers/arrakis-operations-helper.service.ts

```
@@ -35,12 +38,26 @@ export class ArrakisOperationsHelperService {
     await this.vaultDbService.saveCollectedFeeEvent(collctedFeeEvent);
   }
 
+  public async checkIfEntryExists(log: SingleLogEvent) {
+    const eventId = (log.transactionHash + log.logIndex).toLowerCase();
+    try {
+      const exists = await this.vaultDbService.checkIfEntryExists(eventId);
+      return exists;
+    } catch (error) {
+      throw error;
```

Biome

[error] The catch clause that only rethrows the original error is redundant.

These unnecessary catch clauses can be confusing. It is recommended to remove them.

(lint/complexity/noUselessCatch)